packages/plugin-fuse/src/actions/createToken.ts

```diff
@@ -94,21 +75,43 @@ export const createTokenAction = {
         callback?: any
     ) => {
         try {
-            const privateKey = runtime.getSetting(
-                "EVM_PRIVATE_KEY"
-            ) as `0x${string}`;
+            const privateKey = runtime.getSetting("EVM_PRIVATE_KEY") as `0x${string}`;
             const walletProvider = new WalletProvider(privateKey);
-            const action = new QueueAction(walletProvider);
-            return await action.queue(options);
-        } catch (error) {
-            console.error("Error in vote handler:", error.message);
+            const action = new CreateTokenAction(walletProvider);
+
+            const tokenParams: TokenCreationParameters = {
+                name: options.name,
+                symbol: options.symbol,
+                decimals: options.decimals,
+                initialSupply: options.initialSupply,
+                tokenOwner: options.tokenOwner as `0x${string}`,
+                factoryAddress: options.factoryAddress as `0x${string}`,
+                fromChain: options.fromChain,
+            };
+
+            const result = await action.create(tokenParams);
```

Comment:
```suggestion
            const result = await action.create(options);
```

Reply:
this avoid to create tokenParams from options so it simplifies the modifications by removing many lines above.